src/Microsoft.Health.Dicom.Core/Features/Partitioning/Partition.cs

```diff
@@ -0,0 +1,39 @@
+﻿// -------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
+// -------------------------------------------------------------------------------------------------
+
+using System;
+using System.Text.Json.Serialization;
+using EnsureThat;
+using Newtonsoft.Json;
+
+namespace Microsoft.Health.Dicom.Core.Features.Partitioning;
+
+public class Partition
+{
+    public const string DefaultName = "Microsoft.Default";
+
+    public const string UnknownName = "Unknown"; // use for older binaries where we are unable to get partition name and it is not known
+
```

Comment:
Not sure why we want Unknown name. It should always be Microsoft.Default. There is no entry in DB that corresponds to Unknown Partition entry